include/gba/m4a_internal.h

```diff
@@ -334,7 +292,47 @@ struct Song
 extern const struct MusicPlayer gMPlayTable[];
 extern const struct Song gSongTable[];
 
+#define MAX_DIRECTSOUND_CHANNELS 12
```

Comment:
Why move this definition?

Reply:
So it can be grouped together with the PCM_DMA_BUF_SIZE, as both macros are used in conjunction a lot